miniconf/load_site_data.py

```diff
@@ -287,11 +289,20 @@ def normalize_track_name(track_name: str) -> str:
     return track_name
 
 
+def get_card_image_path_for_paper(paper_id: str, paper_images_path: str) -> str:
+    if os.path.exists(os.path.join(paper_images_path, f"{paper_id}.png")):
+        return f"{paper_images_path}/{paper_id}.png"
+    else:
+        print(f"WARNING: using default image for {paper_id}")
+        return f"{paper_images_path}/default.png"
```

Comment:
This partially address https://github.com/acl-org/acl-2020-virtual-conference/issues/174 (cc @prabhat1081 )